packages/components/src/List/ListComposition/Manager/hooks/useCollectionFilter.hook.js

```diff
@@ -1,20 +1,31 @@
-import { useMemo, useState } from 'react';
+import { useCallback, useState } from 'react';
 import isNil from 'lodash/isNil';
 
+/**
+ * By default, filter is case insensitive and without accents
+ * @param value Raw cell value
+ * @param textFilter User input using for filtering
+ * @returns {boolean} if input matches cell content
+ */
 function defaultFilterFunction(value, textFilter) {
```

Comment:
Not sure it will work if the `textFilter` contains accent.
If you want to filter on "brulée" and you remove all the accents from the `value`. You will never have any result 🤔